circuit_maintenance_parser/provider.py

```diff
@@ -173,7 +173,7 @@ def get_default_include_filters(cls) -> Dict[str, List[str]]:
             return cls._include_filter.get_default()  # type: ignore
```

Comment:
does this need ()?

Reply:
no, the `()` (instantiating the class) is necessary in the pydantic 1.x case but appears not to be in the pydantic 2.x case.